Remove deprecated summarizeProtocolTree option [breaking]

pragya91 opened the pull request:
Removed deprecated ILoaderOption = summarizeProtocolTree.
--Closing pr for another clean one.

pragya91 committed:
Remove deprecated summarizeProtocoltree option

markfields commented:
How much of this can you check in now, without including the actual break?  e.g. the changes in `packages/test/test-service-load/src/optionsMatrix.ts` look like they can happen first? To leave this break PR as minimal as possible.

jason-ha commented:
This and #23355 appear to be duplicates. (I just added review for #23355 a few minutes ago.)

jason-ha commented:
> This and #23355 appear to be duplicates. (I just added review for #23355 a few minutes ago.)

Not a duplicate - #23355 is the non-breaking portion of this work. (The changes to remove deprecated use.) That should be completed before this change.

pragya91 committed:
compat broken

dannimad committed:
skip r11s driver on load offline with blob redirect table test (#24042)

Skipping due to flaky R11s fetch error. 

`Error:
R11s fetch error: {"message":"Error when processing POST request to
/repos/frs/e7236aab-132d-4ae7-a39e-fadfb15cfc1e/git/summaries?initial=false","canRetry":false,"isFatal":true,"source":"[post]
request to [http://frs-historian-gitrest] failed with [400] status
code"} Error: R11s fetch error: {"message":"Error when processing POST
request to
/repos/frs/e7236aab-132d-4ae7-a39e-fadfb15cfc1e/git/summaries?initial=false","canRetry":false,"isFatal":true,"source":"[post]
request to [http://frs-historian-gitrest] failed with [400] status
code"} at createGenericNetworkError
(file:///mnt/vss/_work/1/test/node_modules/@fluidframework/driver-utils/src/network.ts:236:9)
at createR11sNetworkError
(file:///mnt/vss/_work/1/test/node_modules/@fluidframework/routerlicious-driver/src/errorUtils.ts:141:12)
at throwR11sNetworkError
(file:///mnt/vss/_work/1/test/node_modules/@fluidframework/routerlicious-driver/src/errorUtils.ts:154:8)
at RouterliciousStorageRestWrapper.request
(file:///mnt/vss/_work/1/test/node_modules/@fluidframework/routerlicious-driver/src/restWrapper.ts:280:3)
at processTicksAndRejections (node:internal/process/task_queues:95:5) at
WholeSummaryUploadManager.writeSummaryTree
(file:///mnt/vss/_work/1/test/node_modules/@fluidframework/routerlicious-driver/src/wholeSummaryUploadManager.ts:28:14)
at Function.timedExecAsync
(file:///mnt/vss/_work/1/test/node_modules/@fluidframework/routerlicious-driver/node_modules/@fluidframework/telemetry-utils/src/logger.ts:715:16)
at WholeSummaryDocumentStorageService.uploadSummaryWithContext
(file:///mnt/vss/_work/1/test/node_modules/@fluidframework/routerlicious-driver/src/wholeSummaryDocumentStorageService.ts:232:25)
at ContainerRuntime.submitSummary
(file:///mnt/vss/_work/1/test/node_modules/@fluidframework/container-runtime/src/containerRuntime.ts:3930:14)
at SummaryGenerator.summarizeCore
(file:///mnt/vss/_work/1/test/node_modules/@fluidframework/container-runtime/src/summary/summaryGenerator.ts:320:18)`
